pkg/server/commands/acquire.go

```diff
@@ -15,19 +16,35 @@ import (
 )
 
 type acquireCmd struct {
-	context      *gin.Context
-	clientset    *ofcirclientv1.OfcirV1Client
-	namespace    string
-	resourceType ofcirv1.CIResourceType
+	context       *gin.Context
+	clientset     *ofcirclientv1.OfcirV1Client
+	namespace     string
+	resourceTypes []ofcirv1.CIResourceType
 }
 
-func NewAcquireCmd(c *gin.Context, clientset *ofcirclientv1.OfcirV1Client, ns string, resourceType string) command {
+func NewAcquireCmd(c *gin.Context, clientset *ofcirclientv1.OfcirV1Client, ns string, resourceType_str string) command {
+	// type can be a comma seperated list
+	resourceTypes_split := strings.Split(resourceType_str, ",")
+	resourceTypes := make([]ofcirv1.CIResourceType, len(resourceTypes_split))
+	for i, v := range resourceTypes_split {
+		resourceTypes[i] = ofcirv1.CIResourceType(v)
+	}
+
 	return &acquireCmd{
-		context:      c,
-		clientset:    clientset,
-		namespace:    ns,
-		resourceType: ofcirv1.CIResourceType(resourceType),
+		context:       c,
+		clientset:     clientset,
+		namespace:     ns,
+		resourceTypes: resourceTypes,
+	}
+}
+
+func contains(rtypes []ofcirv1.CIResourceType, rtype ofcirv1.CIResourceType) bool {
```

Comment:
nit: what about using a map instead (tip: could be populated while parsing happens)